rasa/shared/core/training_data/structures.py

```diff
@@ -169,11 +170,16 @@ def _bot_string(story_step_element: Event) -> Text:
         return f"    - {story_step_element.as_story_string()}\n"
 
     @staticmethod
-    def _or_string(story_step_element: Sequence[Event], e2e: bool) -> Text:
+    def _or_string(story_step_element: Sequence[Event], e2e: bool) -> Optional[Text]:
         for event in story_step_element:
-            if not isinstance(event, UserUttered):
+            # OR statement can also contain `slot_was_set`, and
+            # we're going to ignore this events when representing
```

Comment:
Only trying to understand why is it better to ignore the `SlotSet` event?

Reply:
fixed that, was a stupid assumption

Reply:
Is this comment incorrect now then?